README.md

```diff
@@ -4,3 +4,14 @@ Hi, my name is Eunbae Jeon and I am 24 years old.
 I have been studying front end engineering for about a year part-time.
 If you want to check out my previous projects, you can do it at my github repo.
 Thank you for reading my introduction.
+
+
+
+
+## 자기소개
+
+- 안녕하세요. 프론트엔드 개발자 정재윤입니다.
+- 서울에 살고 있습니다.
+- git flow 협업 연습하고 있습니다. 감사합니다.
+
```

Comment:
Please add one skill that you want to master as a practice.

Reply:
No problem.

Reply:
Well done!